src/collections/_documentation/data-management/rollups.md

```diff
@@ -24,14 +51,14 @@ The first and most important part is that Sentry only groups by stack trace fram
 
 Depending on the information the following data can be used for each stack trace frame:
 
--   Module name
--   Normalized filename (removed from revision hashes etc.)
--   Normalized context line (essentially a cleaned up version of the sourcecode of the affected line if provided)
+- Module name
+- Normalized filename (removed from revision hashes etc.)
+- Normalized context line (essentially a cleaned up version of the sourcecode of the affected line if provided)
 
 This grouping usually works well, but causes two annoying artifacts if not dealt with:
 
--   Minimized JavaScript sourcecode will destroy the grouping in really bad ways. Because of this you should ensure Sentry can access your [Source Maps]({%- link _documentation/clients/javascript/sourcemaps.md -%}#raven-js-sourcemaps).
--   If you modify your stack trace by introducing a new level through the use of decorators, your stack trace will change and so will the grouping. For this matter many SDKs support hiding irrelevant stack trace frames. For instance the Python SDK will skip all stack frames with a local variable called `__traceback_hide__` set to _True_).
+- Minimized JavaScript sourcecode will destroy the grouping in really bad ways. Because of this you should ensure Sentry can access your [Source Maps]({%- link _documentation/clients/javascript/sourcemaps.md -%}#raven-js-sourcemaps).
```

Comment:
This link probably wants to be updated to the new SDK:
`https://docs.sentry.io/platforms/javascript/#source-maps`